composer.json

```
@@ -59,7 +59,7 @@
         "knplabs/knp-menu-bundle": "^2.1 || ^3.0",
         "knplabs/knp-time-bundle": "^1.10",
         "lcobucci/jwt": "^3.4 || ^4.0",
-        "league/commonmark": "^1.6",
+        "league/commonmark": "^2.0@dev",
```

```suggestion
        "league/commonmark": "^2.0@RC",
```
Using `@RC` should be possible here I think, and should be better compatible with installing the Contao RC version.